pkg/dockerregistry/server/quotarestrictedblobstore.go

```diff
@@ -58,7 +99,7 @@ var _ distribution.BlobStore = &quotaRestrictedBlobStore{}
 func (bs *quotaRestrictedBlobStore) Put(ctx context.Context, mediaType string, p []byte) (distribution.Descriptor, error) {
 	context.GetLogger(ctx).Debug("(*quotaRestrictedBlobStore).Put: starting")
 
-	if err := admitBlobWrite(ctx, bs.repo); err != nil {
+	if err := admitBlobWrite(ctx, bs.repo, int64(len(p))); err != nil {
```

Comment:
No, it is designed for a small objects like manifest. Layers are written with a [blobWriter](https://github.com/openshift/origin/pull/8195/files/009f353ce0f5825f8e3821be026ae9f687ab66dc#diff-e95ee3281aac32429f16cc1573f4e64cR152). Actually, this check is useless because we don't store manifests in registry's storage. I'll remove it.